project.clj

```diff
@@ -47,108 +31,27 @@
                  ;; 'uberjar-exclusions' example toward the end of this file.
                  [org.jruby/jruby-stdlib "1.7.20.1"]
 
-                 ;; we do not currently use this dependency directly, but
-                 ;; we have documentation that shows how users can use it to
-                 ;; send their logs to logstash, so we include it in the jar.
-                 ;; we may use it directly in the future
-                 [net.logstash.logback/logstash-logback-encoder "4.5.1"]
-
-
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/trapperkeeper-authorization "0.5.0"]
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/ssl-utils "0.8.1"]
-                 [puppetlabs/dujour-version-check "0.1.2" :exclusions [org.clojure/tools.logging]]
-                 [puppetlabs/http-client "0.5.0"]
-                 [puppetlabs/comidi "0.3.1"]]
-
-  :main puppetlabs.trapperkeeper.main
-
-  :pedantic? :abort
 
-  :source-paths ["src/clj"]
-  :java-source-paths ["src/java"]
-  :test-paths ["test/unit" "test/integration"]
-  :resource-paths ["resources" "src/ruby"]
-
-  :repositories [["releases" "http://nexus.delivery.puppetlabs.net/content/repositories/releases/"]
-                 ["snapshots" "http://nexus.delivery.puppetlabs.net/content/repositories/snapshots/"]]
-
-  :plugins [[lein-release "1.0.5" :exclusions [org.clojure/clojure]]]
+                 [org.clojure/tools.logging "0.3.1"]
```

Comment:
Just curious why you went with this version.  Looks like the version that the latest code in trapperkeeper and kitchensink use is version 0.2.6.